projects/core/src/occ/adapters/configurator/variant/occ-configurator-variant.adapter.ts

```diff
@@ -187,7 +184,6 @@ export class OccConfiguratorVariantAdapter
       tap((resultConfiguration) => {
         resultConfiguration.owner = {
           ...parameters.owner,
-          hasObsoleteState: false,
         };
       })
     );
```

Comment:
Shouldn`t we use MAP instead of TAP?

Reply:
Correct..

Reply:
I need to replace all 8 occurrences of tap here